src/hooks/mobile.tsx

```diff
@@ -12,8 +12,14 @@ const enum MobileSearchParams {
 export const useMobileAppLogin = () => {
   const [searchParams] = useSearchParams();
   const challenge = searchParams.get(MobileSearchParams.CHALLENGE);
+  if (challenge !== undefined && challenge !== null) {
```

Comment:
I think you could simplify?

```suggestion
  if (challenge) {
```

Reply:
Hmmm ... Not sure, it would also check that is is not an empty string ... Maybe ?